### PR TITLE
objects block russian description Close #647

### LINKED ISSUE #647: objects russian description

### PR DESCRIPTION
**Pull request by Murdalay**

### CODE REVIEW

File: common.blocks/objects/objects.ru.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# objects
+
+Блок `objects` предоставляет набор методов-хелперов для работы с объектами JavaScript.
+
+Блок реализован в технологии `vanilla.js` и подходит для использования в серверных и клиентских приложениях.
```
Comment (tadatuta) [inline common.blocks/objects/objects.ru.md]: «как на клиенте, так и на сервере»

File: common.blocks/objects/objects.ru.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+## Публичные методы блока
+
+Блок предоставляет следующий набор публичных метода:
```
Comment (tadatuta) [inline common.blocks/objects/objects.ru.md]: зачем эта фраза, если есть заголовок?

File: common.blocks/objects/objects.ru.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# objects
```
Comment (tadatuta) [inline common.blocks/objects/objects.ru.md]: не хватает примеров

**Comment by Murdalay** [comment]
It seems that yo have closed the wrong issue. Pull request was about    BEMHTML modes order but you closed the object description issue.

**Comment by tadatuta** [comment]
@Murdalay Branch was named `issues/#647@v2` and that's why https://github.com/bem/bem-core/issues/647 was closed. I think it's better to update the name of this issue to fit real changes and create new one about `objects`.

**Pull request merged**